codalab/apps/analytics/views.py

```diff
@@ -19,13 +19,21 @@ def analytics_detail(request):
     for date in User.objects.all().dates('date_joined', 'month'):
         if date.year not in monthly_total_users_joined:
             monthly_total_users_joined[date.year] = {}
-        users_joined_this_month = User.objects.filter(date_joined__month=date.month,
-                                                      date_joined__year=date.year).aggregate(Count('pk'))
-        monthly_total_users_joined[date.year][date.strftime("%B")] = users_joined_this_month['pk__count']
+        users_joined_this_month = User.objects.filter(
+            date_joined__month=date.month,
+            date_joined__year=date.year).aggregate(
+            Count('pk'))
+        monthly_total_users_joined[
+            date.year][
+            date.strftime("%B")] = users_joined_this_month['pk__count']
```

Comment:
I know you are changing this so it is pep8, but i think 80 line count is a bit much it was easier to understand and read before.  In my linter I run local i have error `E501` off "line too long

https://github.com/jcrocholl/pep8/blob/master/pep8.py#L198

this also applies to the render change below.